snips/Dockerfile

```diff
@@ -1,11 +1,14 @@
-ARG BUILD_FROM
-FROM $BUILD_FROM
+#ARG BUILD_FROM
+#FROM $BUILD_FROM
+FROM snipsdocker/platform:x86-0.51.14
 
 # Copy data
 COPY run.sh /
 COPY mosquitto.conf /etc/
 COPY asoundrc /root/.asoundrc
 
+ADD https://github.com/tschmidty69/hassio-snips/releases/download/0.1-pre1/assistant.zip /assistant-default.zip
```

Comment:
Use a build argument for the version with `build.json`: https://home-assistant.io/developers/hassio/addon_config/#add-on-extended-build

Reply:
Got it. Can be changed to point to a release in this repo if you want to upload it.

Reply:
You can upload the zip into this folder as a file.